projects/wave-core/src/lib/operators/dialogs/data-repository/raster/source-dataset.component.html

```diff
@@ -16,12 +16,16 @@
     </mat-chip-list>
 
     <mat-list>
+        <mat-list-item *ngIf="dataset?.time_start || dataset?.time_end">
+            <p><b>Time validity:</b> {{dataset?.time_start.toString()}} - {{dataset?.time_end.toString()}}</p>
```

Comment:
```suggestion
            <p><b>Time validity:</b> {{dataset?.time_start}} - {{dataset?.time_end}}</p>
```
I think `toString()` is called automatically in templates, so it is redundant and you do it differently below.

Reply:
fixed in 900ce5db69b7712d277d6400055ede30c92368c1